mobile/native/src/db/mod.rs

```diff
@@ -0,0 +1,58 @@
+pub mod models;
```

Comment:
🙃 I think we usually leave the use statements at the top.

I think it usually goes:

```rust
use foo;

mod bar;

pub use baz;

const  QUX;
```